src/api/rest/server/mcis/control.go

```diff
@@ -326,6 +325,81 @@ func RestPostMcisRecommend(c echo.Context) error {
 	return c.JSON(http.StatusCreated, content)
 }
 
+// RestGetControlMcis godoc
+// @Summary Control the lifecycle of MCIS
+// @Description Control the lifecycle of MCIS
+// @Tags [MCIS] Control lifecycle
+// @Accept  json
+// @Produce  json
+// @Param nsId path string true "Namespace ID"
+// @Param mcisId path string true "MCIS ID"
+// @Param action query string false "Action to MCIS" Enums(status, suspend, resume, reboot, terminate)
```

Comment:
```suggestion
// @Param action query string false "Action to MCIS" Enums(suspend, resume, reboot, terminate, refine)
```

- status 도 이 함수에서 처리하게 되는지요?
- refine 은 어디에 보내는게 좋을까요?

Reply:
336 라인에서 `status, ` 부분은 삭제하는 것이 맞겠습니다. 😊

As of 2021-09-15:

- `refine` 기능은 `/ns/{nsId}/mcis/{mcisId} [get]` API에 들어 있습니다 😊
- `suspend, resume, reboot, terminate` 기능이 `/ns/{nsId}/mcis/{mcisId} [get]` API에도 남아 있습니다.
  제거해도 괜찮다면 (cb-webtool, CB-MCKS 에 안내 등), 제거하면 좋겠네요 😊

---
 
- `/ns/{nsId}/mcis/{mcisId} [get]` API에는 정보/상태 조회 기능 (metadata, status, id, simple) 만 남긴다면,
  `refine` 기능은 다른 API로 옮기는 것이 좋을 것 같습니다. 😊
  - 그런데, `/ns/{nsId}/control/mcis/{mcisId} [get]` 은 약간.. 라이프사이클 컨트롤 API 같아 보이는 저의 고정관념이 있어서
    `refine` 기능을 `/ns/{nsId}/control/mcis/{mcisId} [get]` API로 옮기는 것이 적절한지는 저는 약간 고민스럽네요.. 🤔